doc/api_methods.md

```diff
@@ -530,9 +550,13 @@ Sets sublayer parameters. Useful when more than one parameter needs to be change
 
 #### Arguments
 
-- **layerDefinition**: an object with the sql and cartocss that defines the data, like:
+Arguments | Description
+--- | ---
+layerDefinition | an object with the sql and cartocss that defines the data
+
+
+#### layerDefinition
```

Comment:
I'd remove this section since we have an example below:
![screen shot 2015-11-04 at 11 39 55](https://cloud.githubusercontent.com/assets/390398/10935840/08178454-82e9-11e5-9431-706b96098db7.png)